features/testbot/traffic_turn_penalties.feature

```diff
@@ -47,7 +47,7 @@ Feature: Traffic - turn penalties applied to turn onto which a phantom node snap
             | 1    | e  | ab,be,be | 36 km/h  | 30s +-1 |
             | b    | f  | bc,cf,cf | 36 km/h  | 40s +-1 |
             | 2    | f  | bc,cf,cf | 36 km/h  | 30s +-1 |
-            | c    | g  | cd,dg,dg | 144 km/h | 10s +-1 |
+            | c    | g  | cd,dg,dg | 72 km/h  | 20s +-1 |
```

Comment:
The use of negative turn penalties is the cause of the increase in speed/time here.

Previously, the start would snap to `cf` (which would be combined with `cd` in the route result annotation). 
Now with multiple intersection candidates to choose from, the route does actually start from `cd`. 

There is [special case logic](https://github.com/Project-OSRM/osrm-backend/blob/919fe74c4064cd3de05b73be873b35c2d75a5052/include/engine/routing_algorithms/routing_base.hpp#L322) for preventing negative weights/durations on the first turn. Despite the intention of this test, it didn't apply previously when starting at `cf` as the negative weight was on the second turn. 
This leads to the multiple candidate route actually having larger weight/duration than previously.

I can see historical [discussions](https://github.com/Project-OSRM/osrm-backend/issues/3683) about the value of negative turn penalty weights, but I'm not sure if that also makes sense for negative durations too. In any case, I've just updated the expected value to reflect this first turn clamping logic.